### ISSUE TITLE
→ Accessibility: Components | CVV Field

### ISSUE THREAD

**Issue by aweell** [Telefonica/mistica-design#1563]
Epic #557 

## Description

The CVV Field since includes a tooltip, has a different approach than the rest of the input fields.

* When using the mouse, the tooltip should be visible in hover
* When using the keyboard the tooltip must be visible when focusing the field, not the tooltip trigger, and the content of the tooltip must be linked to the field using `aria-labelledby`

**APG documentation**

Doc: https://www.w3.org/WAI/ARIA/apg/patterns/tooltip/
Example: https://codepen.io/Moiety/pen/LaPvWy

## Tasks

- [ ] Update specs
- [ ] Design documentation
- [ ] Add to mobile library
- [ ] Add to desktop library
- [ ] Add to UI Kit
- [ ] Android ticket
- [ ] iOS ticket
- [ ] Web ticket
